### DISCUSSION TITLE
Android Play Store / iPhone App

### DISCUSSION THREAD

**Discussion by HackersCardgame** [jamulussoftware/jamulus#1971] [Ideas]
Halli Hallo @corrados 

ist mal noch vorgesehen das in den Google Playstore zu nehmen? Afaik ist da ein Jahres Abo um zu publishen irgend 25€, könnte ich sonst übernehmen.

Dann gibt's eigentlich ne iPhone App?

Gruss
Marc jr. Landolt
eidg. dipl. Informatiker HF
Neuenburgerstrasse 6
5004 Aarau

**Comment by ann0see** [comment]
Hi @HackersCardgame,
The project has changed a lot and corrados is now no longer active in development (only hosting directory servers) and I'm moving over to the documentation. 

Yes, there is an [iPhone](https://github.com/jamulussoftware/jamulus/pull/1875) (working, smaller issues will probably never get it into the AppStore due to a lot of work on the GUI needed and [maybe this legal concern](https://github.com/jamulussoftware/jamulus/pull/1874)) and [Android](https://jamulus.io/wiki/Installation-for-Android) app available. 
If you want to move this forward, feel free to comment on the relevant PR for iOS and contact the person behind https://github.com/jamulussoftware/jamulus/pull/1528 and emlynmac (iOS) and start working on it.

Actually, you could also think about the following:

You might want to look at including [Jamulus for Android in FDroid](https://f-droid.org/en/docs/Inclusion_How-To/) which is probably easier for now than the PlayStore. Of course, the play store version can follow.

Please keep in mind that the mobile versions also need stable internet connectivity (5G/Ethernet, not WiFi)

**Comment by ann0see** [comment]
Hi, I just opened a request for FDroid: 

https://gitlab.com/fdroid/rfp/-/issues/2011#note_837311154

**Reply by ann0see** [reply]
And there‘s a PR for ipa generation #2355 if we’re lucky TestFlight (iOS beta App Store) might have us sometime.

**Comment by ann0see** [comment]
iOS: https://github.com/orgs/jamulussoftware/discussions/3595
If you'd still like to handle the Android distribution, that would be great!
